### ISSUE TITLE
Add release badge

### ISSUE THREAD

**Issue by Runixe786** [ziadOUA/m3-Markdown-Badges#42]
Will be great to have a dynamic latest release badge in material you style.Awesome project btw.

**Comment by DeiseFreire** [comment]
@ziadOUA Your art is beautiful. Simply enchanting.

**Comment by DeiseFreire** [comment]
> @Runixe786 "Awesome project btw".

I agree, amazing project. Congratulations to the project creator @ziadOUA❤️

**Comment by ziadOUA** [comment]
> Will be great to have a dynamic latest release badge in material you style.Awesome project btw.

I guess it would display the latest version ? I'll see what I can do

**Comment by Runixe786** [comment]
Yes exactly, latest version badge. Thanks, waiting for it
